tests/jax/components/building/networks_test.py

```diff
@@ -0,0 +1,209 @@
+from typing import Any, Callable, Dict, List, Sequence
+
+import jax
+import pytest
+
+from mava.components.jax.building import DefaultNetworks
+from mava.components.jax.building.networks import Networks, NetworksConfig
+from mava.core_jax import SystemBuilder
+from mava.specs import MAEnvironmentSpec
+from mava.systems.jax import Builder
+from tests.jax.mocks import make_fake_env_specs
+
+
+@pytest.fixture
+def test_network_factory() -> Callable:
+    """Pytest fixture for network factory.
+
+    Returns:
+        Network factory using custom make_default_networks.
+    """
+
+    def make_default_networks(
```

Comment:
@mmorris44 Can't we use the `make_default_networks` that exist in ppo already?

Reply:
Thanks for the review @KaleabTessera. Yes we could, but I don't want the tests to be using something external to the code I'm trying to test here. I.e. if someone changes the `make_default_networks` of PPO, these tests should not break. Thus, I defined my own version here to show how the function might be defined in the wild.

Reply:
Cool. Makes sense.